app/assets/css/inspector/_juju-inspector.scss

```diff
@@ -596,10 +596,10 @@
             .header-slot,
             .viewlet-manager-navigation,
             .viewlet-manager-footer {
-                @include flex-basis(auto);
```

Comment:
Are `flex-basis()` and `flex()` used anywhere else in the app? Are they causing issues there? Why does this fix the problem? :)